ui/src/features/stage/create-stage.tsx

```diff
@@ -109,7 +115,11 @@ export const CreateStage = ({
   const onSubmit = handleSubmit(async (data) => {
     let value = data.value;
     if (tab === 'wizard') {
-      const unmarshalled = stageFormToYAML(watch(), project || '');
+      const unmarshalled = stageFormToYAML(
+        watch(),
```

Comment:
This watch here looks bad I think. The intention of usage for this feature is different IMO.

Reply:
Agreed. Now It uses `getValues()`